composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "stefanorg/graphql-middleware",
+    "name": "bertrandgauthier/graphql-middleware",
```

Comment:
there is not need to do this. Composer handle this when you include your forked version read this: http://stackoverflow.com/a/13500676